src/formatter.ts

```diff
@@ -263,6 +263,11 @@ export default class Formatter {
         .replace(
           /(?:{{--\s*?blade-formatter-disable-next-line\s*?--}}|{{--\s*?prettier-ignore\s*?--}}|<!--\s*?prettier-ignore\s*?-->)[\r\n]+[^\r\n]+/gis,
           (match: any) => this.storeIgnoredLines(match),
+        )        
+        // comment ignore
+        .replace(
+          /(?:^{{--\s)(?:.|[\n\r])*--}}/gis,
+          (match: any) => this.storeIgnoredLines(match),
```

Comment:
@Dave-iFour 
It seems when a comment is present multiple times, the text in between is also subject to the regular expression. 

e.g. `bar` is also ignored

```blade
{{-- 
foo
--}}

bar

{{--
baz
--}}
```

I've added a fix below where it is more appropriate place. Can you add a test in `__tests__/formatter.ts`?